bakery/env/local.json

```diff
@@ -5,5 +5,9 @@
   "S3_ACCESS_KEY_ID": "",
   "S3_SECRET_ACCESS_KEY": "",
   "S3_DIST_BUCKET": "",
-  "VERSIONED_FILE": ""
+  "S3_VERSIONED_BUCKET": "",
```

Comment:
Rename to `QUEUE_STATE_BUCKET`? :stuck_out_tongue:

Reply:
Yeah, let's make this line up with the CF code so it's clear. I'll use your suggested values for this and `VERSIONED_FILE`.